docs/content/toolbar.md

```diff
@@ -74,3 +74,10 @@ Path to your SpatialOS Unreal GDK snapshot.
 
 * **Snapshot file name**</br>
 Name of your SpatialOS Unreal GDK snapshot file.
+
+* **Singleton classes**</br>
+Specifies which classes that should be generated as singleton classes during the [ICG process](./interop.md).
+
+* **Classes to generate typebindings for**</br>
+Specifies the classes for which typebindings should be generated for durint the [ICG process](./interop.md).
```

Comment:
Specifies the classes for which typebindings should be generated **during** the ...
(also no `for` before `during`)

Reply:
Same as above - does the user add these here or do the specifying? If so, we can make it clear they have to do something. We can also simplify the sentence:

```
Use this to specify the classes which the [ICG](./interop.md) should generate [type](./glossary.md#type-bindings) for.
```